### PR TITLE
Complete Changelog and bump version to 1.15.0

### PR DESCRIPTION
**Pull request by claudep**

### CODE REVIEW

File: CHANGELOG.md
Comment (jaap3) [inline CHANGELOG.md:6]: Should this refer to PR #500? Also, isn't this a breaking change? It seems that with this addition it's not longer possible for optional 2FA (i.e. all users are now required to setup a 2FA device).

File: CHANGELOG.md
Comment (jaap3) [inline CHANGELOG.md:7]:
```suggestion
```

File: CHANGELOG.md
Comment (jaap3) [inline CHANGELOG.md:9]: Maybe nicer to group these together

```suggestion
- Confirmed Python 3.11 support
- Confirmed Django 4.1 support
```

File: CHANGELOG.md
Comment (claudep) [reply CHANGELOG.md:6]: If you experimented that optional 2FA is no longer possible, could you please create a new ticket for that?

Comment (jaap3) [reply CHANGELOG.md:6]: I think I misunderstood the purpose of that PR, seems everything is working as before

Comment (claudep) [reply CHANGELOG.md:6]: phew… :smile:

**Comment by claudep** [comment]
@jaap3, may I ask you to approve the PR, if not already done, so as I can push to master?

**Review by jaap3 — APPROVED** [review]

**Comment by claudep** [comment]
Thanks, but seemingly not sufficient for pushing, probably because you are not Jazzband member?

**Comment by pickfire** [comment]
@moggers87 sorry for the ping but I wonder if we can have a version bump?

**Comment by matthewhegarty** [comment]
+1 for a version bump please 🙏 , currently blocked from upgrading to django 4.1 due to #512.

**Comment by pickfire** [comment]
IIRC I tested the latest git commit and have some issues with migrations, but I don't recall what was it, so I didn't use the git version either.

**Comment by matthewhegarty** [comment]
If it helps, I just tried a build of my app which includes `django-two-factor-auth` master.  I can see there is a new migration ([0008_delete_phonedevice.py](https://github.com/jazzband/django-two-factor-auth/blob/master/two_factor/migrations/0008_delete_phonedevice.py)).  This migration applied ok.  

All my tests passed (including with upgrade to django 4.1), and my application code (including 2FA) seems to be working correctly.  So from my testing this release is ok.

**Comment by claudep** [comment]
This PR is still missing an approval by a Jazzband member.

**Comment by moggers87** [comment]
The Python 3.11 build is failing for some reason, though I can't see why.

**Comment by moggers87** [comment]
I was looking in the wrong place:

```
 ======================================================================
  Error: ERROR: test_webauthn_attestation_and_assertion (two_factor.plugins.webauthn.tests.test_e2e.E2ETests.test_webauthn_attestation_and_assertion)
  ----------------------------------------------------------------------
  Traceback (most recent call last):
    File "/home/runner/work/django-two-factor-auth/django-two-factor-auth/./two_factor/plugins/webauthn/tests/test_e2e.py", line 96, in test_webauthn_attestation_and_assertion
      self.register_authenticator()
    File "/home/runner/work/django-two-factor-auth/django-two-factor-auth/./two_factor/plugins/webauthn/tests/test_e2e.py", line 82, in register_authenticator
      self.wait_for_url(urljoin(self.base_url, reverse("two_factor:setup")))
    File "/home/runner/work/django-two-factor-auth/django-two-factor-auth/./two_factor/plugins/webauthn/tests/test_e2e.py", line 65, in wait_for_url
      WebDriverWait(self.webdriver, self.timeout).until(EC.url_to_be(url))
    File "/home/runner/work/django-two-factor-auth/django-two-factor-auth/.tox/py311-dj32-webauthn/lib/python3.11/site-packages/selenium/webdriver/support/wait.py", line 90, in until
      raise TimeoutException(message, screen, stacktrace)
  selenium.common.exceptions.TimeoutException: Message: 
```

Not sure why this is happening now though as it seems #567 was fine

**Comment by moggers87** [comment]
@claudep I made a separate PR to fix the webauthn tests: https://github.com/jazzband/django-two-factor-auth/pull/582

**Review by moggers87 — APPROVED** [review]
LGTM

**Pull request merged**
